packages/twenty-server/src/engine/core-modules/workspace/crons/jobs/check-custom-domain-valid-records.cron.job.ts

```diff
@@ -39,7 +39,7 @@ export class CheckCustomDomainValidRecordsCronJob {
 
     for (const workspace of workspaces) {
       try {
```

Comment:
Should we also check all public domains for that workspace?

Also @AMoreaux how did you address my feedback on rate-limiting last time? Any input here to guide Martin based on the strategy you had adopted? This will come even sooner as now one customer can add many domains